static/src/js/util/request/admin/__tests__/metricsPreferencesRequest.test.js

```diff
@@ -0,0 +1,50 @@
+import MetricsPreferencesRequest from '../metricsPreferencesRequest'
+import Request from '../../request'
+
+beforeEach(() => {
+  jest.restoreAllMocks()
+  jest.clearAllMocks()
+})
+
+describe('metricsPreferencesRequest#constructor', () => {
+  test('sets the default values when authenticated', () => {
+    const token = '123'
+    const request = new MetricsPreferencesRequest(token)
+
+    expect(request.authenticated).toBeTruthy()
+    expect(request.authToken).toEqual(token)
+    expect(request.baseUrl).toEqual('http://localhost:3000')
+  })
+})
+
+describe('metricsPreferencesRequest#transformResponse', () => {
```

Comment:
This is not a function on the `MetricsPreferencesRequest` class, it can just be handled by the test in the parent class